### DISCUSSION TITLE
Wizard Form under the same URL

### DISCUSSION THREAD

**Discussion by turkle12** [react-hook-form/react-hook-form#3369] [Q&A] [answered]
Hi,

Is it possible to use the Wizard under one url, for example, `example.com/apply` and move through the steps, instead of pushing the user to a new route like `example.com/apply/step1`? 

Thanks!

**Comment by bluebill1049** [comment, marked as answer]
you can do that 100%, just remember to host a global state for the different sections of your forms.

**Reply by turkle12** [reply]
Great, thanks @bluebill1049
